pkg/controller/certificates/trigger/trigger_controller.go

```diff
@@ -219,14 +228,19 @@ func (c *controller) updateOrApplyStatus(ctx context.Context, crt *cmapi.Certifi
 	}
 }
 
-// shouldBackoffReissuingOnFailure tells us if we should back-off re-issuing for
-// an hour or not. Notably, it returns no back-off when the certificate doesn't
+// shouldBackOffReissuingOnFailure returns true if an issuance needs to be
+// delayed and the required delay after calculating the exponential backoff.
+// The backoff periods are 1h, 2h, 4h, 8h, 16h, 32h counting from when the
+// failure occured,
```

Comment:
```suggestion
// most recent failure occurred,
```

(I still need to convince folks to accept check-spelling somehow...)